common/src/cams/test-utilities/offices.mock.ts

```diff
@@ -3504,3 +3504,4 @@ export const OFFICES: OfficeDetails[] = [
 
 export const MANHATTAN = OFFICES.find((office) => office.courtDivisionCode === '081');
 export const BUFFALO = OFFICES.find((office) => office.courtDivisionCode === '091');
+export const DELAWARE = OFFICES.find((office) => office.courtDivisionCode === '111');
```

Comment:
We will probably have to update these when we get a final answer on which offices serve which court division